app/code/Magento/Sales/Model/Order/Config.php

```diff
@@ -156,12 +159,13 @@ public function getStatusLabel($code)
      * Retrieve status label for area
      *
      * @param string|null $code
+     * @param int|null $storeId
      * @return string|null
      * @since 102.0.1
      */
-    public function getStatusFrontendLabel(?string $code): ?string
+    public function getStatusFrontendLabel(?string $code, int $storeId = null): ?string
```

Comment:
Unfortunately, we cannot change the signature of the public method in a class marked as @api.